runtime/include/stdchpl.h

```diff
@@ -2,15 +2,15 @@
  * Copyright 2020-2021 Hewlett Packard Enterprise Development LP
  * Copyright 2004-2019 Cray Inc.
  * Other additional copyright holders may be indicated within.
- *
+ * 
  * The entirety of this work is licensed under the Apache License,
  * Version 2.0 (the "License"); you may not use this file except
  * in compliance with the License.
- *
+ * 
```

Comment:
The blank spaces coming back in this file and Makefile.share below is a little unfortunate.

Reply:
Sure, but `git revert` / `patch` aren't smart enough for that and given the rush to get this done last night after the lack of volunteers, I felt it was below my pay grade to waste more time / sleeping time on.  I think this is not the only file that has this "issue" for people who are worried about this sort of thing (and my sense, based on recent PRs, is that there's some editor which some devs use which strips trailing whitespaces automatically, such that these "improvements" are spread through tons of PRs rather than being tackled once and for all..?).